containers/Dockerfile.base-spack

```diff
@@ -15,10 +15,13 @@ LABEL au.org.access-nri.ci.spack-repo-version ${SPACK_REPO_VERSION}
 # Install spack
 RUN git clone -c feature.manyFiles=true https://github.com/spack/spack.git $SPACK_ROOT --branch ${SPACK_REPO_VERSION} --single-branch --depth=1
 
+RUN sed -i 's/granularity: microarchitectures/granularity: generic/' $SPACK_ROOT/etc/spack/defaults/concretizer.yaml
```

Comment:
Hey, would you be able to give me an explanation of this line? I'm not too familiar with what its doing to the concretizer.

Reply:
https://spack.readthedocs.io/en/latest/build_settings.html#selection-of-the-target-microarchitectures

We had to change the default from microarchitectures because the instructions being used were architecture specific and different from the COSIMA build of ACCESS-OM2.